### PR TITLE
fix: Adjust release pipeline to be created as pre-release instead of latest

### PR DESCRIPTION
**Pull request by gzukel**
# Description

fix: Adjust release pipeline to be created as pre-release instead of latest

Closes: DEVOP-567

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested via Github Actions.

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [x] Tested via GitHub Actions 

# Checklist:

- [N/A] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: changelog.md
Comment (kingpinXD) [inline changelog.md:58]:
```suggestion
* [1940](https://github.com/zeta-chain/node/pull/1940) - adjust release pipeline to be created as pre-release instead of latest
```

**Review by kingpinXD — APPROVED** [review]
lgtm , lets wait for an approval from someone from devops as well

**Review by billyyang423 — APPROVED** [review]

**Pull request merged**
